src/transmission_manager.rs

```diff
@@ -1,15 +1,13 @@
 use ixa::{
-    context::Context,
-    define_person_property, define_person_property_with_default, define_rng,
-    error::IxaError,
-    global_properties::ContextGlobalPropertiesExt,
-    people::{ContextPeopleExt, PersonId, PersonPropertyChangeEvent},
-    random::ContextRandomExt,
+    define_person_property, define_person_property_with_default, define_rng, Context,
+    ContextGlobalPropertiesExt, ContextPeopleExt, ContextRandomExt, IxaError, PersonId,
+    PersonPropertyChangeEvent,
 };
 use statrs::distribution::{ContinuousCDF, Exp, Poisson};
 
+use crate::contact::ContextContactExt;
```

Comment:
Nit: possible to make one `use crate::{}` import statement? I find that easier to read.